tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
14.04 hasn't even letsencrpyt as it seems. In this case users simply need to set `certbot_install_from_source: yes` and rely on the Git installation. That's what it's good for :)